Examples/JamitFoundationExample/App/Sources/TableView/TableViewController.swift

```diff
@@ -0,0 +1,51 @@
+//  Copyright © 2020 Jamit Labs GmbH. All rights reserved.
+
+import Foundation
+import JamitFoundation
+import UIKit
+
+class TableViewController: StatefulViewController<TableViewViewModel> {
```

Comment:
Shouldn't we use ListViewController instead? Or should this be the example implementation for a TableViewController with an StatefulViewController?

Reply:
This is the example for a `TableView` usage where we use different types of `ContainerTableViewCell`s. With a `ListViewController` you can only show one type of table view cell.

Reply:
Ok I got it 👍